debian/postinst

```diff
@@ -66,7 +77,11 @@ case "$1" in
       # CACHE_DIR is no longer present or used since 19.1
       rm -rf /var/cache/ubuntu-advantage-tools
 
-      grep -iq trusty /etc/os-release && configure_esm
+      if [ "14.04" = "$VERSION_ID" ]; then
+        configure_esm
+      else
+        unconfigure_esm
```

Comment:
Why the unconfigure_esm addition?

Reply:
Only install ESM sources.list on package install for Trusty, not on any other release
So, we actually don't want to advertize any esm packages on non-trusty in general.

Reply:
The point is we didn't have `unconfigure_esm` before. Are you thinking about release upgrades here, i.e., when going from trusty to xenial sometime in the future?

Reply:
Ops, now I actually read your response fully.